packages/kit/test/apps/options-paths-base/svelte.config.js

```diff
@@ -0,0 +1,29 @@
+/** @type {import('@sveltejs/kit').Config} */
+const config = {
+	extensions: ['.jesuslivesineveryone', '.whokilledthemuffinman', '.svelte.md', '.svelte'],
+	kit: {
+		files: {
+			assets: 'public',
+			lib: 'source/components',
+			routes: 'source/pages',
+			template: 'source/template.html'
+		},
+		// appDir: '_wheee',
+		floc: true,
+		target: '#content-goes-here',
+		host: 'example.com',
+		trailingSlash: 'always',
+		vite: {
+			build: {
+				minify: false
+			},
+			clearScreen: false
+		},
+		paths: {
+			base: '/path-base',
```

Comment:
Would it be possible to just set this in the existing options test? I think Rich had some concern about adding tests for new apps because it might slow things down

Reply:
Done